docs/shared-backend/README.md

```diff
@@ -18,7 +18,7 @@ If you're aware of stand-alone MinIO set up, the installation and running remain
 
```

Comment:
I think this must be deprecated as well, right @harshavardhana ?

Reply:
If so, I suggest handling deprecation in a separate PR.
Leaving for now, as the intent of this PR is simply fix the URLs to the docs site and not to refactor the whole Github documentation.